engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/RuntimeTypesTest.scala

```diff
@@ -0,0 +1,533 @@
+package org.enso.interpreter.test.instrument
+
+import org.apache.commons.io.output.TeeOutputStream
+import org.enso.interpreter.runtime.EnsoContext
+import org.enso.interpreter.runtime.`type`.ConstantsGen
+import org.enso.interpreter.test.Metadata
+import org.enso.polyglot._
+import org.enso.polyglot.runtime.Runtime.Api
+import org.enso.text.editing.model
+import org.enso.text.editing.model.TextEdit
+import org.graalvm.polyglot.Context
+import org.scalatest.BeforeAndAfterEach
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import java.io.{ByteArrayOutputStream, File}
+import java.nio.file.{Files, Paths}
+import java.util.UUID
+
+@scala.annotation.nowarn("msg=multiarg infix syntax")
+class RuntimeTypesTest
```

Comment:
I'm not sure it is worth it but we already have runtime instrumentation tests in Java